package.json

```diff
@@ -34,16 +34,16 @@
     "index"
   ],
   "scripts": {
-    "test": "npm run lint && tap test/{unit,acceptance}/{*,**/*}.test.js && npm run test:types",
-    "test:unit": "tap test/unit/{*,**/*}.test.js",
+    "test": "npm run lint && tap test/{unit,acceptance}/{*,**/*,**/**/*}.test.js && npm run test:types",
```

Comment:
Looks like this already got merged in PR #294?

Reply:
yes, I included that commit cause I needed this command to run the tests in this branch.

see in the PR description:

> (includes https://github.com/opensearch-project/opensearch-js/pull/294 cause I needed it to run the tests)

Reply:
rebased with main 👌🏻